pipeline/transformer/transformer_sierra/src/main/scala/uk/ac/wellcome/platform/transformer/sierra/transformers/SierraContributors.scala

```diff
@@ -37,82 +46,92 @@ object SierraContributors
       marcTag = marcTag,
       marcSubfieldTags = List("a", "b", "c", "d", "e", "t", "0")
-    )
-
-    persons
-      .flatMap { subfields: List[MarcSubfield] =>
-        val hasSubfieldT = subfields.exists {
-          _.tag == "t"
-        }
-        val roles = getContributionRoles(subfields)
+    ).flatMap { subfields: List[MarcSubfield] =>
+      val hasSubfieldT = subfields.exists {
+        _.tag == "t"
+      }
+      val roles = getContributionRoles(subfields)
 
-        val maybeAgent = if (hasSubfieldT) {
-          getLabel(subfields)
-            .map {
-              Agent(_)
-            }
-            .map { agent =>
-              identify(subfields, agent, "Agent")
-            }
-        } else {
-          getPerson(subfields, normalisePerson = true)
-            .map { person =>
-              identify(subfields, person, "Person")
-            }
-        }
+      val maybeAgent = if (hasSubfieldT) {
+        getLabel(subfields)
+          .map {
+            Agent(_)
+          }
+          .map { agent =>
+            identify(subfields, agent, "Agent")
+          }
+      } else {
+        getPerson(subfields, normalisePerson = true)
+          .map { person =>
+            identify(subfields, person, "Person")
+          }
+      }
 
-        maybeAgent map { agent =>
-          Contributor(
-            agent = agent,
-            roles = roles
-          )
-        }
+      maybeAgent map { agent =>
+        Contributor(
+          agent = agent,
+          roles = roles
+        )
       }
-  }
+    }
 
   /* For a given MARC tag (110 or 710), return a list of all the Contributor[Organisation] instances
    * this MARC tag represents.
    */
   private def getOrganisationContributors(
     bibData: SierraBibData,
-    marcTag: String): List[Contributor[MaybeDisplayable[Organisation]]] = {
-    val organisations = getMatchingSubfieldLists(
+    marcTag: String): List[Contributor[MaybeDisplayable[Organisation]]] =
+    getMatchingSubfieldLists(
       bibData,
       marcTag = marcTag,
       marcSubfieldTags = List("a", "b", "c", "d", "e", "0")
-    )
+    ).flatMap { subfields: List[MarcSubfield] =>
+      val roles = getContributionRoles(subfields)
+      val maybeAgent = getOrganisation(subfields)
+
+      maybeAgent.map { agent =>
+        Contributor(
+          agent = identify(subfields, agent, "Organisation"),
+          roles = roles
+        )
+      }
+    }
 
-    organisations
-      .flatMap { subfields: List[MarcSubfield] =>
-        val roles = getContributionRoles(subfields)
-        val maybeAgent = getOrganisation(subfields)
+  private def getMeetingContributors(
+    bibData: SierraBibData,
+    marcTag: String): List[Contributor[MaybeDisplayable[Meeting]]] = {
+    getMatchingSubfieldLists(
+      bibData,
+      marcTag = marcTag,
+      marcSubfieldTags = List("a", "c", "d", "j", "t", "0")
+    ).flatMap { subfields: List[MarcSubfield] =>
+      val roles = getContributionRoles(subfields, "j")
```

Comment:
[This in the spec](https://www.loc.gov/marc/bibliographic/bd111.html) says "Relator term" <= is that a role?

Reply:
Sorry, me being silly, missed it on Person...